config/routes.rb

```diff
@@ -6,6 +6,7 @@
   end
 
   resources :whitehall_assets, only: %i(create)
+  get '/whitehall_assets/*path' => 'whitehall_assets#show'
```

Comment:
Did you consider using the standard resource-based route for `show`? If you did, it would be useful to explain why you didn't use it in the commit note. I have to admit it seems a bit odd to me, but I'm probably missing something!

Reply:
I've added the following to the commit message to explain this:

> Requests look something like '/whitehall_assets/government/uploads/asset.png'. I can't use the standard Rails restful route for the show action because that doesn't appear to work for `:id` parameters that include slashes.